src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+/*
+ * Represents a person's block number in the address book
+ */
+
+public class Block {
+	String block = new String();
+	
+	public Block() {
```

Comment:
Unnecessary since empty constructors are provide by default. However, think about whether it would be better to have your own constructor that takes in an argument instead